stylesheets/themes/tennessee/theme.scss

```diff
@@ -41,24 +41,35 @@ $footerLightColor: $theme-main-color;
 $footerDarkColor: $mobileNavColor;
 
 .homeLogo {
-  width: auto !important;
+  width: 100% !important;
   & h1 {
-    margin-left: 0 !important;
+    margin: auto !important;
+    text-align: center;
   }
 }
 
 .localLogo {
   visibility: hidden;
+  display: none;
```

Comment:
Do you need both `visibility: hidden` and `display: none`?  I think that just `display: none` will give the same effect, but that's worth double-checking.

Reply:
Probably not. You're right.